src/adapter/mysql.cr

```diff
@@ -76,6 +76,31 @@ class Granite::Adapter::Mysql < Granite::Adapter::Base
     end
   end
 
+  def import(table_name, primary_name, primary_auto, fields, model_array)
+  statement = String.build do |stmt|
+    stmt << "INSERT INTO #{quote(table_name)} ("
+    stmt << "#{quote(primary_name)}, " unless primary_auto
+    stmt << fields.map { |field| quote(field) }.join(", ")
+    stmt << ") VALUES "
+
+    model_array.each do |model|
+      if model.responds_to? :to_h
+        next unless model.valid?
+        stmt << "("
+        stmt << model.to_h[primary_name].to_s + ", " unless primary_auto
+        stmt << fields.map { |field| model.to_h[field].is_a?(String) ? "'" + model.to_h[field].to_s + "'" : model.to_h[field] }.join(", ")
```

Comment:
I'm not a fan of how I'm doing this.  Suggestions welcome!

Reply:
The adapters now provide [a quote method](https://github.com/amberframework/granite-orm/blob/master/src/adapter/base.cr#L66-L69) but the functionality is has some bugs.

Reply:
The quoting method I do use for column/table names. However, since it quotes with the backtick that couldn't be used for string values afaik?

Reply:
That was my initial thought, however when I tried it I kept getting like `db params invalid format` or something like that (not home atm to look).  The params should just be an array of values, which the first value of array maps to first ? placeholder correct?